packages/mui-material/src/Badge/Badge.js

```diff
@@ -7,13 +7,15 @@ import { unstable_composeClasses as composeClasses } from '@mui/base/composeClas
 import { useBadge } from '@mui/base/useBadge';
 import { useSlotProps } from '@mui/base';
 import { styled } from '../zero-styled';
-import useThemeProps from '../styles/useThemeProps';
+import { createUseThemeProps } from '../zero-useThemeProps';
```

Comment:
You can just re-export from the existing zero-styled file. Otherwise we'll have to modify the plugin code everytime we add new API.